core/modules/worker_split.h

```diff
@@ -7,6 +7,8 @@ class WorkerSplit final : public Module {
  public:
   static const gate_idx_t kNumOGates = Worker::kMaxWorkers;
 
+  WorkerSplit() { max_allowed_workers_ = kNumOGates; }
```

Comment:
It is a variable declared in the base class, and cannot be initialized in the initializer list of a derived class, see for [example](http://stackoverflow.com/questions/7405740/how-can-i-initialize-base-class-member-variables-in-derived-class-constructor).